refactor/config_data/params.json

```diff
@@ -30,5 +66,26 @@
   "delta_EMB": {
       "description": "Ratio between durability and current sea temperature (unit: days / °C)",
       "value": 210
+  },
+  "infection_main_delta": {
```

Comment:
Currently some deltas are listed twice under different names (this one corresponds to `delta_CO_0`). Would be good to fix it.